src/protect-api/README.md

```diff
@@ -1,30 +1,54 @@
+languages:
+- csharp
+page_type: sample
+name: ASP.NET Core minimal web API that protects API
+description: This ASP.NET Core minimal web API protects an API endpoint. The code in this sample is used by one or more articles on docs.microsoft.com.
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-csharp
+---
+# ASP.NET Core minimal web API - Protects API | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
 
 The sample code provided here has been created using minimal web API in ASP.NET Core 6.0, and slightly modified to be protected for a single organization using [ASP.NET Core Identity](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity?view=aspnetcore-6.0) that interacts with [Microsoft Authentication Library (MSAL)](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-overview).  In other words, a very minimalist web api is secured by adding an authorization layer before user requests can reach protected resources.  At this point it is expected that the user sign-in had already happened, so api calls can be made in the name of the signed-in user. For that to be possible a token containing user's information is being sent in the request headers and used in the authorization process.
 
-```output
-┌──────────────────────────────────┐                               ┌─────────────────────────────┐
-│                                  │                               │                             │
-│   ASP.NET Core 6                 │        Http Request           │  ASP.NET Core 6             │
-│                                  ├──────────────────────────────►│                             │
-│   Web app that signs in users    │  Authorization Bearer 1NS...  │  Protected Minimal web Api  │
-│                                  │  with forescast.read scope    │                             │
-└──────────────────────────────────┘                               └─────────────────────────────┘
+<!-- TODO: IMAGE or CONSOLE OUTPUT of running/executed app -->
 
-Scenario:
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
 
-An (ASP.NET Core) web app that allows users to sign in enables the possibility of acquiring and validating their tokens for specific audiences and scopes.
+## Prerequisites
 
-Later the web app can make calls to protected Apis in the name of the signed-in users.
-```
+1. An Azure Active Directory (Azure AD) tenant. You can [open an Azure account for free](https://azure.microsoft.com/free) to get an Azure AD instance.
+1. [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
```

Comment:
I think Marsh requested these be an unordered list.

Reply:
thanks for feedback @ckittel,

done | addressed from [bed1e](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/8/commits/bed1e42ab9247a88ee347495fe6a58c4672b63f5)